definitions/procedures/backup/offline/foreman_db.rb

```diff
@@ -31,21 +31,42 @@ def run
       private
 
       def local_backup
-        with_spinner("Collecting data from #{pg_data_dir}") do
-          feature(:foreman_database).backup_local(
-            pg_backup_file,
-            :listed_incremental => File.join(@backup_dir, '.postgres.snar'),
-            :volume_size => @tar_volume_size,
-            :data_dir => pg_data_dir
-          )
+        with_spinner("Collecting data from #{pg_data_dir.join(',')}") do
+          pg_data_dir.each_with_index do |pg_dir, index|
+            do_backup(pg_dir, 'create') if index == 0
+            do_backup(pg_dir, 'append') if index != 0
+          end
         end
       end
 
+      def do_backup(pg_dir, cmd)
+        feature(:foreman_database).backup_local(
+          pg_backup_file,
+          :listed_incremental => File.join(@backup_dir, '.postgres.snar'),
+          :volume_size => @tar_volume_size,
+          :data_dir => pg_dir,
+          :command => cmd
+        )
+      end
+
       def pg_backup_file
         File.join(@backup_dir, 'pgsql_data.tar')
       end
 
       def pg_data_dir
```

Comment:
If you're changing this from a single value to multiple I'd let the name reflect that:
```suggestion
      def pg_data_dirs
```